scripts/populate_preprint_providers.py

```diff
@@ -65,16 +65,14 @@ def update_or_create(provider_data):
         return new_provider, True
 
 def main(env):
-    use_plos = '--plos' in sys.argv
-
-
     PREPRINT_PROVIDERS = {
```

Comment:
This should eventually be moved to either a [django fixture](https://docs.djangoproject.com/en/1.10/howto/initial-data/) or a django migration that uses the loaddata functionality.